app/src/main/webapp/js/controllers.js

```diff
@@ -489,7 +489,14 @@ esbMessageAdminControllers.controller('SyncCtrl', ['$scope', '$rootScope', 'EsbM
     };
 
     $scope.sync = function() {
-        alert("create sync req:"+$scope.syncEntity.value+" "+$scope.syncSystem.value+" "+$scope.syncKey.value+" "+$scope.syncValues);
+
+        var values = '';
+        for (var value in $scope.syncValues) {
+            values+=$scope.syncValues[value]+",";
+        }
+        EsbMessageService.sync($scope.syncEntity.value, $scope.syncSystem.value, $scope.syncKey.value, values).then(function(response){
+            alert("successfull!");
```

Comment:
Sounds good, thanks for adding the issue to track this enhancement (https://github.com/esbtools/esb-message-admin/issues/26)